Chinese/DesertHell/NewWorld.txt

```diff
@@ -0,0 +1,4 @@
+This is a new world
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.